var/spack/repos/builtin/packages/umt/package.py

```diff
@@ -0,0 +1,196 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import socket
+from os import environ as env
+
+from spack.package import *
+
+class Umt(CachedCMakePackage):
+    """Umt is a LLNL mini-app based on the Teton thermal radiative transport library."""
+
+    homepage = "https://github.com/LLNL/UMT"
+    url = ""
+    git = 'https://github.com/LLNL/UMT.git'
+
+    version("develop", branch="develop", submodules=False)
+    maintainers = ["aaroncblack"]
+
+    # The CMakeLists.txt is in 'src' directory.
+    root_cmakelists_dir = "src"
+
+    ###########################################################################
+    # package variants
+    ###########################################################################
+
+    variant("openmp", default=True, description="Enable OpenMP support")
+    variant("openmp_offload", default=False, description="Enable OpenMP target offload support")
+    variant("caliper", default=False, description="Enable Caliper performance timers")
+    variant("mpi", default=True, description="Enable MPI support (mandatory")
+    variant("mfem", default=False, description="Enable support for reading MFEM meshes")
+    variant("umpire", default=False, description="Enable use of Umpire memory library")
+
+    ###########################################################################
+    # package dependencies
+    ###########################################################################
+
+    #######################
+    # CMake
+    #######################
+    depends_on("cmake@3.13.3:", type="build")
+
+    #######################
+    # Dependencies
+    #######################
+    depends_on("mpi")
```

Comment:
```suggestion
    depends_on("mpi", when="+mpi")
```
I'm assuming you don't need it when MPI is off

Reply:
MPI is mandatory.

Reply:
@aaroncblack then we should remove the mpi variant above